app/src/docs/v1.api-spec.yaml

```diff
@@ -1093,52 +1119,10 @@ components:
       bearerFormat: JWT
     OpenID:
       type: openIdConnect
-      openIdConnectUrl: 'https://example.com/.well-known/openid-configuration'
```

Comment:
I'd leave this line in - it's useful in the event this spec yaml is somehow parsed and taken from source instead of from hosted to have this be populated. Granted, value isn't that useful, but gives the reader a clue that it's a standard well known config to look for.